cpufreq@mtwebster/files/cpufreq@mtwebster/applet.js

```diff
@@ -92,13 +94,20 @@ function rd_nums_frm_file(file) {
 }
 function num_to_freq_panel(num) {
     num = Math.round(num);
-    if (num < 1000)
-        return num + ' kHz';
-    if (num < 1000000)
-        return Math.round(num / 10) / 100 + ' MHz';
-    if (num < 1000000000)
-        return Math.round(num / 10000) / 100 + ' GHz';
-    return Math.round(num / 10000000) / 100 + ' THz';
+    let units;
+    if (num < 1000) {
+        units = 'kHz';
+    } else if (num < 1000000) {
+        num = Math.round(num / 10) / 100;
+        units = 'MHz';
+    } else if (num < 1000000000) {
+        num = Math.round(num / 10000) / 100;
+        units = 'GHz';
+    } else {
+        num = Math.round(num / 10000000) / 100;
+        units = 'THz';
+    }
+    return formatNum(num) + ' ' + units;
```

Comment:
There is a built-in function for this instead.
```suggestion
    return num.toFixed(2) + ' ' + units;
```